src/xdocs/config_annotation.xml

```diff
@@ -1355,15 +1355,16 @@ package com.example.annotations.packageannotation; //ok
       </subsection>
 
       <subsection name="Examples" id="SuppressWarningsHolder_Examples">
-        <p>To prevent <code>FooCheck</code> violations from being reported write:</p>
-        <source>
-@SuppressWarnings("foo") interface I { }
-@SuppressWarnings("foo") enum E { }
-@SuppressWarnings("foo") InputSuppressWarningsFilter() { }
-        </source>
         <p>Some real check examples:</p>
         <p>This will prevent from invocation of the MemberNameCheck:</p>
```

Comment:
>This will prevent from invocation of the

Please make it `This will prevent selected violations from`

Reply:
done